.circleci/config.yml

```diff
@@ -72,6 +70,7 @@ jobs:
               echo "Waiting for RskJ..."
               sleep 1
             done
+            node generateBtcBlocks.js  
```

Comment:
I think I'd prefer running this right after `npm install`, but I'm fine either way.